lib/msal-browser/src/app/ClientApplication.ts

```diff
@@ -54,6 +54,9 @@ export abstract class ClientApplication {
     // Callback for subscribing to events
     private eventCallbacks: Map<string, EventCallbackFunction>;
 
+    // Redirect Response Object
+    private redirectResponse: Promise<AuthenticationResult | null> | undefined;
```

Comment:
If the page is re-rendered won't this variable be re-rendered also? Doesn't this only solve `handleRedirectPromise` being called multiple times in a single page load?

Reply:
Yes, that's right. More specifically this solves `handleRedirectPromise` being called multiple times per pca instance.

Reply:
in the PR description it says that it may solve issues when frameworks re-render components, we should update this in case someone gets confused thinking this will solve re-rendered instances.

Reply:
For a min, I thought the same. :-D

Reply:
Page refresh is not a broken scenario. This is only to guard against concurrent calls to handleRedirectPromise throwing errors